notebooks/15_flightplanning_tuto.ipynb

```diff
@@ -0,0 +1,1018 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Flight Planning\n",
+    "\n",
+    "The problem we are targetting is to predict the trajectory of an aircraft under forecasted weather conditions.\n",
+    "\n",
+    "This problem has been chosen for two reasons:\n",
+    "  - Show how scikit-decide can be used in close to real life use case,\n",
+    "  - Design a specific domain using external packages. In our case, we will use [OpenAP](https://github.com/TUDelft-CNS-ATM/openap) to simulate the dynamics of the aircraft.\n",
+    "\n",
+    "Therefore in this notebook we will go through the following steps:\n",
+    "  - Wrap a Gym environment in a scikit-decide domain;\n",
```

Comment:
Not Done in this notebook (is it a copy paste mistake ?)

Reply:
Yes will fix